data-prepper-expression/src/test/java/org/opensearch/dataprepper/expression/OperatorProviderTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.expression;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class OperatorProviderTest {
+
+    @Mock
+    Operator<Boolean> testOperator;
+
+    OperatorProvider objectUnderTest;
+
+    @Test
+    void testGetExistingOperator() {
+        final int testSymbol = 100;
+        when(testOperator.getSymbol()).thenReturn(testSymbol);
+        final List<Operator<?>> testOperators = Collections.singletonList(testOperator);
```

Comment:
I'd probably put a few operators in this list just to be sure it works with multiple.